.gitignore

```diff
@@ -273,4 +273,6 @@ dist
 .yarn/unplugged
 .yarn/build-state.yml
 .yarn/install-state.gz
-.pnp.*
\ No newline at end of file
+.pnp.*
+
+package-lock.json
```

Comment:
Should not be ignored. package-lock.json describes the npm modules tree and makes it easier to reproduce. We need it for CI/CD pipelines.